recipes/phytop/setup.py

```
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+from src.__version__ import version
+
+from setuptools import setup, find_packages
+from distutils.extension import Extension
```

Ruff

`distutils.extension.Extension` imported but unused

Remove unused import: `distutils.extension.Extension`

(F401)